src/Styleguide/Pages/Artwork/ArtworkApp.tsx

```diff
@@ -0,0 +1,207 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { FullArtworkGrid } from "Styleguide/Components/ArtworkGrid"
+import { Tab, Tabs } from "Styleguide/Components/Tabs"
+import { Col, Row } from "Styleguide/Elements/Grid"
+import { Boot } from "Styleguide/Pages/Boot"
+import { exhibitions } from "Styleguide/Pages/Fixtures//SelectedExhibitions"
+import { bio } from "Styleguide/Pages/Fixtures/ArtistBio"
+import { imageData } from "Styleguide/Pages/Fixtures/Artwork/imageData"
+import { RegularArtwork2Editions } from "Styleguide/Pages/Fixtures/Artworks"
+import { insights } from "Styleguide/Pages/Fixtures/MarketInsights"
+import { AboutTheWork } from "./Components/AboutTheWork"
+import { ArtistInfo } from "./Components/ArtistInfo"
+import { Banner } from "./Components/Banner"
+import { Bibliography } from "./Components/Bibliography"
```

Comment:
since there are no sub-routes on artist (the tabs don't deep-link) i've dropped most of the components into `/Components`. This will keep things clean should new routes eventually be added.